padguidedb/padguidedb.py

```diff
@@ -111,40 +118,74 @@ async def searchdungeon(self, ctx, *, search_text):
             for page in pagify(msg):
                 await ctx.send(box(page))
 
-    @padguidedb.command()
-    @checks.is_owner()
-    async def setdungeonscriptfile(self, ctx, *, dungeon_script_file):
-        """Set the dungeon script file."""
-        self.settings.setDungeonScriptFile(dungeon_script_file)
-        await ctx.tick()
+    @padguidedb.group()
+    async def series(self, ctx):
+        """Series"""
```

Comment:
what happens when you call 

`^pdb series` by itself? is there a default command help list? how would that tie into the help usage for each `@series.command(name="...")` below?

Reply:
It shows subcommands.  The docstring there doesn't affect anything other than the help string shown when you run ^pdb series.